specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/loadBalancer.json

```diff
@@ -1159,6 +1159,10 @@
           "$ref": "./network.json#/definitions/IPAllocationMethod",
           "description": "The Private IP allocation method."
         },
+        "privateIPAddressVersion": {
+          "$ref": "./network.json#/definitions/IPVersion",
+          "description": "Available from Api-Version 2019-04-01 onwards, it represents whether the specific ipconfiguration is IPv4 or IPv6. Default is taken as IPv4."
```

Comment:
I don't think you need to mention API version as it's already implied because this property isn't present in older specs.

Reply:
Resolved. Please take a look.